### PR TITLE
doc: metrics

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #1031 

Extended metrics section, but I do not know all the details. As these settings do not seem to be necessary to start working with Artipie, I've added todo to verify them later.

### CODE REVIEW

File: .wiki/Configuration-Metrics.md [hunk 1]
```diff
@@ -1,19 +1,49 @@
...
 
 ```yaml
 meta :
   metrics :
     type : prometheus
-```
\ No newline at end of file
+```
+Metrics in Prometheus compatible format will be available on `http://{host}:{port}/prometheus/metrics` path,
+were `{host}` and `{port}` Artipie service host and port.
```
Comment (dgarus) [inline .wiki/Configuration-Metrics.md:49]: were -> where

File: .wiki/Configuration-Metrics.md [hunk 2]
```diff
@@ -1,19 +1,49 @@
...
+meta:
+  metrics:
+    type: asto # Metrics type, `asto` to publish statistics into storage
+    storage: # Storage to publish the metrics to
+      type: fs
+      path: /tmp/artipie/statistict
+```
+
+Storage metrics can be obtained via HTTP `GET` request by path `http://{host}:{port}/.metrics`, 
+were `{host}` and `{port}` Artipie service host and port accordingly. The response is a json object
```
Comment (dgarus) [inline .wiki/Configuration-Metrics.md:36]: were -> where

**Review by dgarus — CHANGES_REQUESTED** [review]
@olenagerasimova thanks!
There are a few minor comments.

**Comment by olenagerasimova** [comment]
@dgarus thanks, corrected

**Review by dgarus — APPROVED** [review]

**Pull request merged**